infra/examples-dev/gcp-google-workspace/main.tf

```diff
@@ -51,6 +51,7 @@ module "psoxy" {
   jira_server_url                = var.jira_server_url
   jira_cloud_id                  = var.jira_cloud_id
   example_jira_issue_id          = var.example_jira_issue_id
+  deployment_bundle              = var.deployment_bundle
```

Comment:
I know is deprecated, but missing.

Reply:
yeah, fine - the feature just isn't implemented for that case - but OK if it works ...